consensus/state_processing/src/per_block_processing/deneb.rs

```diff
@@ -1,2 +1,9 @@
-#[allow(clippy::module_inception)]
-pub mod deneb;
+use ethereum_hashing::hash_fixed;
+use types::consts::deneb::VERSIONED_HASH_VERSION_KZG;
+use types::{KzgCommitment, VersionedHash};
+
+pub fn kzg_commitment_to_versioned_hash(kzg_commitment: &KzgCommitment) -> VersionedHash {
```

Comment:
moved from `./deneb.rs` it didn't make sense to have that file if this was the only method in it